NuGet.Docs/consume/Videos.cshtml

```diff
@@ -0,0 +1 @@
+@{Response.RedirectPermanent("~/ndocs/consume-packages/videos");}
```

Comment:
yes. This is our new subfolder for all new docs and we have added permanent redirects for everything else.